addon/behaviors/row-expansion.js

```diff
@@ -0,0 +1,23 @@
+import Behavior from 'ember-light-table/behaviors/behavior';
+
+export default Behavior.extend({
+
+  exclusionGroup: 'row-expansion',
+
+  // passed in
+  multiRow: true,
+
+  init() {
+    this._super.call(this, ...arguments);
+    this.events.onToggleExpandedRow = [ 'rowClick:_all' ];
+  },
+
+  onToggleExpandedRow(ltBody, row) {
```

Comment:
I dont think that a behavior should have interaction with the lt-body component. Everything should be done on the table instance which should be then reflected on the required components in the view layer. 

Passing around the component is considered bad practice since it can be destroyed at any time. I would much rather have the table instance be passed around. 

Maybe even just init the behavior with the table instance? That way you only have to do it once. Im not exactly sure of the cleanest route for that though.